### PR TITLE
Added more task to rosa-cross-dc Taskfile.yaml

### PR DESCRIPTION
**Pull request by pruivo**
* route53 -> creates the Route53 DNS records
* monitoring -> installs grafana in both ROSA clusters
* dataset-import -> triggers the user creation in the primary cluster

### CODE REVIEW

File: provision/rosa-cross-dc/Taskfile.yaml [hunk 1]
```diff
@@ -308,3 +307,185 @@ tasks:
...
+    desc: "Install grafana dashboards in both ROSA clusters"
+    deps:
+      - common:split
+      - common:env
+    requires:
+      vars:
+        - ROSA_CLUSTER_NAME_1
+        - ROSA_CLUSTER_NAME_2
+    cmds:
+      - task: install-helms
```
Comment (ahus1) [inline provision/rosa-cross-dc/Taskfile.yaml:419]: AFAIK the `helm` commands also target a specific cluster, so running this without a parameter would target only the currently logged in cluster.

**Review by ahus1 — CHANGES_REQUESTED** [review]
Thank you for this addition. I didn't test it, I just reviewed the code. I found one small thing below, please have a look.

File: provision/rosa-cross-dc/Taskfile.yaml (hunk 1 above)
Comment (pruivo) [reply provision/rosa-cross-dc/Taskfile.yaml]: Those are local helm commands to add the Grafana and Jaeger repositories. I changed the task name to make it more clear.

Comment (ahus1) [reply provision/rosa-cross-dc/Taskfile.yaml]: I didn't know that those command were local only, so I learned something new today!

**Review by ahus1 — APPROVED** [review]

**Pull request merged**